src/qibolab/platform/platform.py

```diff
@@ -170,15 +134,23 @@ def nqubits(self) -> int:
         """Total number of usable qubits in the QPU."""
         return len(self.qubits)
 
+    @property
+    def pairs(self) -> list[QubitPairId]:
+        """Available pairs in thee platform."""
+        return list(self.parameters.native_gates.two_qubit)
+
     @property
     def ordered_pairs(self):
         """List of qubit pairs that are connected in the QPU."""
         return sorted({tuple(sorted(pair)) for pair in self.pairs})
 
     @property
     def topology(self) -> list[QubitPairId]:
-        """Graph representing the qubit connectivity in the quantum chip."""
-        return list(self.pairs)
+        """Graph representing the qubit connectivity in the quantum chip.
+
+        Synonym of :attr:`pairs`.
+        """
+        return self.pairs
```

Comment:
At this stage, we could even drop this in favor of `pairs`.

Reply:
Done in 4f4e6fcc20da4e83f6a0743f093603f3c6949a9f